src/components/DetailsPage/details.css

```diff
@@ -0,0 +1,49 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  background: rgb(219, 215, 223);
+}
+
+.back {
+  width: 48px;
+  height: 33px;
+}
+
+.backBtn {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+  align-items: center;
+}
+
```

Comment:
- [ ] Great job 👍 . Like the previous reviewer mentioned kindly make sure your design looks as close as possible to the original design 🙏 
- [ ] Kindly use the correct font 🙏 
- [ ] Kindly make sure you space out your elements to look as close as possible to the original design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/45231276/197316109-002239d3-0b09-49e7-a0d5-1efbbdc48667.PNG)  |  ![image](https://user-images.githubusercontent.com/74814780/197335863-a48d91ae-8425-4f60-ad74-b75ca0279fa7.png)